test/utility/test_util.h

```diff
@@ -49,9 +49,16 @@
 #include "windows_shim.h"
 #endif
 
-#define TESTUTIL_ENV_CONFIG_TIERED \
-    ",tiered_storage=(bucket=./"   \
-    "bucket,bucket_prefix=pfx-,local_retention=%d,name=%s)"
+#define DIR_STORE_BUCKET_NAME "bucket"
+#define S3_STORE_BUCKET_NAME "s3testext;ap-southeast-2"
```

Comment:
I don't think we should hard code this bucket name if we can avoid it.  One consequence is when it runs in evergreen, it will going to the AsiaPacific for the objects, which isn't what we want to test.  In the `test_s3_connection.cpp` program, the bucket name is given by by the environment variable "WT_S3_EXT_BUCKET", so I suspect that may be set as well as the aws credentials.

Reply:
Fixed in 5dc54e204facff54ad4cadf8cb4524b9e8dc03e7

Reply:
I did not remove #define S3_STORE_BUCKET_NAME but renamed it as S3_DEFAULT_BUCKET_NAME because even in  `test_s3_connection.cpp` the bucket name by default is set to `s3testext` and can be overridden by the environment variable `WT_S3_EXT_BUCKET`. I used the same approach in `testutil_wiredtiger_open` to use the default bucket name if the environment variable is not set.